configure.py

```diff
@@ -2665,6 +2672,9 @@ def find_canonical_os_name(os_name_variant):
     if options.single_amalgamation_file and not options.amalgamation:
         raise UserError("--single-amalgamation-file requires --amalgamation.")
 
+    if options.destdir:
+        raise UserErrror("--destdir was removed. Use the DESTDIR environment variable instead.")
```

Comment:
-r

You can use pylint to find this kind of stuff: `python -m pylint configure.py`